PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -72,13 +73,13 @@ public class PocketSource: Source {
     public convenience init(
         space: Space,
         user: User,
-        sessionProvider: SessionProvider,
+        appSession: AppSession,
         consumerKey: String,
         defaults: UserDefaults,
         backgroundTaskManager: BackgroundTaskManager
     ) {
         let apollo = ApolloClient.createDefault(
-            sessionProvider: sessionProvider,
+            sessionProvider: appSession as! SessionProvider,
```

Comment:
This does not need to be force cast in Services, but for some reason it does here. And it works. @dskuza  may have insights.

Reply:
it feels like we have protocols, types and extensions defined in different modules that also sometimes depend on each other, that might be the cause of this?